### PR TITLE
[WebUI] Add settings.ts, version.ts and SettingsComponent

### LINKED ISSUE #201: [WebUI] Add version string in Settings page

### PR DESCRIPTION
**Pull request by caisq**
Add a settings page. It allows the user to change settings such as TTS voice type and TTS volume. It also contains a 
"Reload" button that allows the user to force-reload the app. See screenshot below:
image

The settings are stored in one of two possible places:
- Via cefsharp.ts saveSettings() and loadSettings() API, which will request the native layer to save/load settings.
  See the updated README.md file
- If cefsharp's bound listener object is unavailable, in browser local storage.

Towards #198: The settings logic is not connected with TextToSpeechComponent yet. The branching of the logic for
personalized (cloud) voice and generic (local) voice will be done in another PR.
Fixes #201: The version string (e.g., v0.0.1) is displayed at the top of the settings page.

Also in this PR: Rearrange the icons in the left nav:
- Move the "Minimize" button to the top. Use the "close" icon
- Put the settings (menu) icon at the bottom.

### CODE REVIEW

**Comment by caisq** [comment]
@antonkast-google Please review this PR. Thanks.

**Review by antonkast-google — APPROVED** [review]

File: webui/README.md [hunk 1]
```diff
@@ -170,3 +172,10 @@ new clickable regions that have appeared since the last call.
 Calling `updateButtonBoxes()` n times with n different `componentName`s will
 cause n sets of clickable regions to be registered.
 
+### 2.4. Saving WebUI user settings to host and loading the settings from host
+
+The functions `saveSettings()` and `loadSettings()` can be used to serialized
+user settings (e.g., TTS voice and volume) to the host and loading the settings
+back from the host. Note that WebViews such as CefShar usually do not persist
```
[the remark this answers is not in the document]
Comment (caisq) [reply webui/README.md:179]: Fixed. Thanks for catching the typo.

File: webui/README.md [hunk 2]
```diff
@@ -170,3 +172,10 @@ new clickable regions that have appeared since the last call.
...
+data stored in `localStorage` after the instance of WebView is destroyed, which
+necessitates settings storage at the host level.
```
[the remark this answers is not in the document]
Comment (caisq) [reply webui/README.md:181]: Good find. Unfortunately this CachePath setting doesn't seem to be present in later versions of CefSharp. We are using 94.4.x right now: 
https://cefsharp.github.io/api/94.4.x/html/R_Project_Documentation.htm

So I'll keep it as is for now.

**Review by caisq — COMMENTED** [review]
Thanks for the review.

**Pull request merged**